docs/tutorial_bracket_strategy.md

```diff
@@ -0,0 +1,224 @@
+
+==> Sending the funds transferring transaction, please execute this transaction second
+
+Signing and posting multi-send transaction 0x09cb78a5a49f10305a2f108d45e8fa059e4231ebf219315c56409d67419d65fb from proposer account 0x740a98F8f4fAe0986FB3264Fe4aaCf94ac1EE96f
+Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
+```
+
+Now, you should just follow the link and sign & execute the transaction with the lower transaction-ID first. It should look like this:
+
+![Read tokenAddressToIdMap](assets/tutorial_bracket_strategy_signing.png)
+
+For executing, just press the button _confirm_.
+
+If you are executing the second transaction with a lower gasPrice than the first one, then you can sign it immediately.
+Otherwise, you should wait until the first one is mined. This works as the gas price usually determines the mining order of the transaction.
+
+Congrats. You managed to provide liquidity and have a chance to earn a passive income, if the prices are moving and returning back to your initial currentPrice.
+
+## Withdrawing liquidity
+
+The withdrawal from the Gnosis Protocol is always done in two step: requesting withdrawals and the actual withdrawal transferring the funds.
+
+### Withdraw request
+
+Withdrawal requests can be made with the following command:
+
+```ssh
+npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma separated brackets]  --tokenIds=[indices] --requestWithdraw --network=$MASTER_SAFE
+```
+
+Here, the flag `--requestWithdraw` is very important part, which tells the script to initiate the the withdraw request.
+The `tokenIndices` of the tokens, which you want to withdraw from the brackets need to be specified via the flag `--tokenIds=[indices]`.
+Under normal usage, these are the exact same indices, which were used during the liquidity provision.
+The flag `--brackets=[comma separated brackets]` needs to contain the brackets that were deployed during the liquidity provision.
+In case you don't remember them, you would have to run the following command:
+
+```ssh
+npx truffle exec scripts/get_deployed_brackets.js --masterSafe=$MASTER_SAFE --network=$NETWORK_NAME
+```
+
+which will display all your previously written brackets and generate a csv listing the brackets with more information.
+
+By running the withdraw request script, you have generated a transaction within your Gnosis-safe interface. Execute this transaction and wait - maximal 5 minutes after the transaction gets mined - until the gnosis protocol closes its batch.
+In the next batch you will be able to withdraw your funds.
+
+### Withdrawing funds
+
+The actual withdraw transaction transferring the tokens back into the MASTER SAFE from the Gnosis Protocol can be initiated with the following command.
+
+```ssh
+npx truffle exec scripts/withdraw.js --masterSafe=$MASTER_SAFE --brackets=[comma separated brackets]  --tokenIds=[indices] --withdraw --network=$MASTER_SAFE
```

Comment:
`--transferFundsToMaster` missing.

Reply:
Also, you might want to specify that the brackets must be comma separated with no spaces after the comma.